### PR TITLE
feat: Automatic style loading

### PR DESCRIPTION
**Pull request by seveibar**
Simplifies API examples because you don't need to import CSS or a font

https://hello-seam.slack.com/archives/C051ETBPRTQ/p1685131057245149

## Future Work

- [ ] Load the relevant version of styles properly from CDN
- [ ] Add content hash to link tag

### CODE REVIEW

File: src/lib/seam/use-seam-styles.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    if (linkElement === null) {
+      const link = document.createElement('link')
+      link.rel = 'stylesheet'
+      link.type = 'text/css'
+      link.href = seamCssUrl
+
+      document.head.appendChild(link)
+    }
+  }, [enabled])
+}
```
Comment (razor-x) [inline src/lib/seam/use-seam-styles.ts:21]:
```suggestion
import { useEffect } from 'react'

// TODO figure out the version automatically
const cssUrl = 'https://www.unpkg.com/@seamapi/react@1.1.0/index.min.css'

export const useSeamStyles = ({ enabled }: { enabled: boolean }) => {
  useEffect(() => {
    if (!enabled) return
    if (globalThis.document == null) return
 
    const linkEl = document.querySelector(`link[href="${seamCssUrl}"]`)

    if (linkEl === null) {
      const link = globalThis.document.createElement('link')
      link.rel = 'stylesheet'
      link.type = 'text/css'
      link.href = cssUrl
      globalThis.document.head.appendChild(link)
    }
  }, [enabled])
}
```

File: .storybook/preview.tsx [hunk 2]
```diff
@@ -66,6 +66,8 @@ const preview: Preview = {
           publishableKey={publishableKey}
           userIdentifierKey={userIdentifierKey}
           endpoint={seamEndpoint}
+          disableCssInjection
+          disableFontInjection
```
Comment (razor-x) [inline .storybook/preview.tsx:70]: Cool. Need to update a few more places though: `README.md`, `examples/basic/App.tsx` (remove css import), `examples/basic/index.html` (remove font import) and disable here: https://github.com/seamapi/react/blob/main/test/fixtures/react.tsx#L18

**Review by razor-x — APPROVED** [review]
Small things, can be done here or another PR.

File: .storybook/preview.tsx (hunk 2 above)
Comment (seveibar) [reply .storybook/preview.tsx:70]: a3917de

File: src/lib/seam/use-seam-font.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    if (linkElement === null) {
+      const link = document.createElement('link')
+      link.rel = 'stylesheet'
+      link.type = 'text/css'
+      link.href = fontUrl
+
+      document.head.appendChild(link)
+    }
+  }, [enabled])
+}
```
[the remark this answers is not in the document]
Comment (seveibar) [reply src/lib/seam/use-seam-font.ts]: a3917de

**Pull request merged**